infra/modules/secret/variables.tf

```diff
@@ -0,0 +1,22 @@
+variable "manage_method" {
+  type        = string
+  description = <<EOF
+    Method to manage the secret. Options are 'manual' or 'code'.
```

Comment:
Nit: I think `manual` vs `generated` might be more intuitive and sets them in opposition to one another. I keep getting confused and thinking that `code` means the secret is stored in a file somewhere.

Reply:
@rocketnova non-urgent in reviewing one of our projects with multiple apps, where an API secret was "generated" in one of the apps that provides a microservice, but "manual" in another app that is a client of the microservice, it makes me think that "manual" isn't really an accurate word in that scenario. "external" is probably more accurate since it's externally defined from the perspective of the app.

kind of nitpicky but curious for your thoughts.